docs/olm.md

```diff
@@ -25,9 +37,6 @@ sudo podman push quay.io/dgoodwin/hive-registry:latest
 
 Create a CatlogSource, OperatorGroup, and Subscription for hive:
 
-
-Create a CatalogSource in your cluster where OLM is running. This will spawn a registry pod using your image just published.
-
 ```
 kubectl apply -f hack/olm-registry/
```

Comment:
The catalog source points to your image https://github.com/openshift/hive/blob/master/hack/olm-registry/catalog-source.yaml#L8. Should that be a template instead?